src/SFA.DAS.AssessorService.Application.Api.Client/ApplyTokenService.cs

```diff
@@ -17,20 +17,22 @@ public ApplyTokenService(IWebConfiguration configuration, IHostingEnvironment ho
 
         public string GetToken()
         {
-            if (_hostingEnvironment.IsDevelopment())
-                return string.Empty;
-
-            var tenantId = _configuration.ApplyApiAuthentication.TenantId;// 
-            var clientId = _configuration.ApplyApiAuthentication.ClientId;// 
-            var appKey = _configuration.ApplyApiAuthentication.ClientSecret;// 
-            var resourceId = _configuration.ApplyApiAuthentication.ResourceId;// 
-
-            var authority = $"https://login.microsoftonline.com/{tenantId}";
-            var clientCredential = new ClientCredential(clientId, appKey);
-            var context = new AuthenticationContext(authority, true);
-            var result = context.AcquireTokenAsync(resourceId, clientCredential).Result;
-
-            return result.AccessToken;
+            return string.Empty;
```

Comment:
Do we not need this token anymore?

Reply:
Not yet. Apply API does not yet have any Authentication. 
This is a stalled branch,
We may have to do progress this for other reasons, but will become redundant when QnA API is ready.